CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Next Release
 -------------
 
+* Remove check for same origin on embedded frames.
```

Comment:
can you be more descriptive? It's fairly clear you are removing the check, but why. Remember, this is open source, so people can't see internal discussions or any comments on your internal ticket that might be helpful as to describing WHY this is being removed.

Reply:
Is the use case that the framed app can navigate to another origin and that origin uses the XFC SDK?

Reply:
@alex-bezek Does the CHANGELOG look okay now ?